CUSTOMISING.md

```diff
@@ -111,6 +111,26 @@ damap:
       db-kind: postgresql # your database type
 ```
 
```

Comment:
placement is good after the database, I would put it in its own title/section
```suggestion

### Custom Database Changes

```

Reply:
Done :)